cmd/catalog-importer/cmd/sync.go

```diff
@@ -125,7 +126,13 @@ func (opt *SyncOptions) Run(ctx context.Context, logger kitlog.Logger, cfg *conf
 
 		syncID, ok := catalogType.Annotations[AnnotationSyncID]
 		if !ok {
-			level.Debug(logger).Log("msg", "ignoring catalog type as it is not managed by an importer")
+			if catalogType.SourceRepoUrl == nil && catalogType.IsEditable {
+				level.Debug(logger).Log("msg", "catalog type is editable and unmanaged",
+					"catalog_type_id", catalogType.Id)
+				unmanagedCatalogTypes = append(unmanagedCatalogTypes, catalogType)
+			} else {
+				level.Debug(logger).Log("msg", "ignoring catalog type as it is not managed by an importer")
```

Comment:
Feels like this should be something like this?
```suggestion
				level.Debug(logger).Log("msg", "ignoring catalog type as it is being managed already")
```